libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -331,8 +331,10 @@ static void _handleConnectFailure( IotMqttConnection_t pMqttConnection,
 /**
  * @brief Place holder packet identifier used when
  * _IotMqtt_AddSubscriptions is called with previous subscriptions lists.
+ * Any non-zero value is acceptable, since this value is never sent out to
```

Comment:
This is a small PR, so not a problem but in general it's best to have single purpose commits.